packaging/Makefile_packaging.mk

```diff
@@ -0,0 +1,516 @@
+# Common Makefile for including
+# Needs the following variables set at a minium:
+# NAME :=
+# SRC_EXT :=
+# SOURCE =
+
+# Put site overrides (i.e. REPOSITORY_URL, DAOS_STACK_*_LOCAL_REPO) in here
+-include Makefile.local
+
+# alternate sources
+#OPENSUSE_MIRROR               ?= https://provo-mirror.opensuse.org
+#OPENSUSE_REPOS_MIRROR         ?= https://opensuse.mirror.liquidtelecom.com
+OPENSUSE_MIRROR               ?= https://download.opensuse.org
+OPENSUSE_REPOS_MIRROR         ?= $(OPENSUSE_MIRROR)
+
+ifeq ($(DEB_NAME),)
+DEB_NAME := $(NAME)
+endif
+
+CALLING_MAKEFILE := $(word 1, $(MAKEFILE_LIST))
+
+DOT     := .
+RPM_BUILD_OPTIONS += $(EXTERNAL_RPM_BUILD_OPTIONS)
+# Find out what we are
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
+# Of course that does not work for SLES-12
+ID := $(shell . /etc/os-release; echo $$ID)
+VERSION_ID := $(shell . /etc/os-release; echo $$VERSION_ID)
+ifeq ($(ID_LIKE),debian)
+UBUNTU_VERS := $(shell . /etc/os-release; echo $$VERSION)
+ifeq ($(VERSION_ID),19.04)
+# Bug - distribution is set to "devel"
+DISTRO_ID_OPT = --distribution disco
+endif
+DISTRO_ID := ubuntu$(VERSION_ID)
+DISTRO_BASE = $(basename UBUNTU_$(VERSION_ID))
+VERSION_ID_STR := $(subst $(DOT),_,$(VERSION_ID))
+endif
+ifeq ($(ID),centos)
+DISTRO_ID := el$(VERSION_ID)
+DISTRO_BASE := $(basename EL_$(VERSION_ID))
+define install_repo
+	if yum-config-manager --add-repo=$(1); then                  \
+	    repo_file=$$(ls -tar /etc/yum.repos.d/*.repo | tail -1); \
+	    sed -i -e 1d -e '$$s/^/gpgcheck=False/' $$repo_file;     \
+	else                                                         \
+	    exit 1;                                                  \
+	fi
+endef
+endif
+ifeq ($(findstring opensuse,$(ID)),opensuse)
+ID_LIKE := suse
+DISTRO_ID := sl$(VERSION_ID)
+DISTRO_BASE := $(basename LEAP_$(VERSION_ID))
+endif
+ifeq ($(ID),sles)
+# SLES-12 or 15 detected.
+ID_LIKE := suse
+DISTRO_ID := sle$(VERSION_ID)
+DISTRO_BASE := $(basename SLES_$(VERSION_ID))
+endif
+ifeq ($(ID_LIKE),suse)
+define install_repo
+	zypper --non-interactive ar $(1)
+endef
+endif
+
+BUILD_OS ?= leap.42.3
+PACKAGING_CHECK_DIR ?= ../packaging
+COMMON_RPM_ARGS := --define "%_topdir $$PWD/_topdir"
+DIST    := $(shell rpm $(COMMON_RPM_ARGS) --eval %{?dist})
+ifeq ($(DIST),)
+SED_EXPR := 1p
+else
+SED_EXPR := 1s/$(DIST)//p
+endif
+SPEC    := $(NAME).spec
+VERSION := $(shell rpm $(COMMON_RPM_ARGS) --specfile --qf '%{version}\n' $(SPEC) | sed -n '1p')
+DEB_VERS := $(subst rc,~rc,$(VERSION))
+DEB_RVERS := $(subst $(DOT),\$(DOT),$(DEB_VERS))
+DEB_BVERS := $(basename $(subst ~rc,$(DOT)rc,$(DEB_VERS)))
+RELEASE := $(shell rpm $(COMMON_RPM_ARGS) --specfile --qf '%{release}\n' $(SPEC) | sed -n '$(SED_EXPR)')
+ifeq ($(SRPM),)
```

Comment:
We shouldn't ever have (want) to have to specify the SRPM manually, so let's not even entertain this being set externally.

Reply:
Missed remove those checks in the raft copy of Makefile_packaging.mk.  Will be changed in next update.